src/models/lineChart.js

```diff
@@ -410,11 +410,14 @@ nv.models.lineChart = function() {
                         allData[indexToHighlight].highlight = true;
                 }
 
+                var defaultValueFormatter = function(d,i) {
+                    return d == null ? "N/A" : yAxis.tickFormat()(d);
+                };
+                var valueFormatter = interactiveLayer.tooltip.valueFormatter() ? interactiveLayer.tooltip.valueFormatter() : defaultValueFormatter;
+
                 interactiveLayer.tooltip
                     .chartContainer(chart.container.parentNode)
-                    .valueFormatter(function(d,i) {
-                        return d === null ? "N/A" : yAxis.tickFormat()(d);
-                    })
+                    .valueFormatter(valueFormatter)
```

Comment:
You can just do this without line 416 there:

    .valueFormatter(interactiveLayer.tooltip.valueFormatter() || defaultValueFormatter);